scripts/o3de/ExportScripts/export_utility.py

```diff
@@ -0,0 +1,195 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+
+import o3de.command_utils as command_utils
+import o3de.export_project as exp
+import o3de.manifest as manifest
+
+from o3de import android, android_support
+
+
+import argparse
+import logging
+import pathlib
+import sys
+
+from typing import List
+
+def process_level_names(ctx, seedfile_paths, level_names, platform):
+    for level in level_names:
+        seedfile_set = set(seedfile_paths)
+        new_level_path = ctx.project_path / f'Cache/{platform}/levels' / level.lower() / (level.lower() + ".spawnable")
+        if new_level_path.is_file() and new_level_path not in seedfile_set:
+            seedfile_paths.append(new_level_path)
+            seedfile_set.add(new_level_path)
```

Comment:
Doing a `new_level_path not in seedfile_set` defeats the purpose of using a set at all. 

```suggestion
        new_level_path = ctx.project_path / f'Cache/{platform}/levels' / level.lower() / (level.lower() + ".spawnable")
        if new_level_path.is_file():
            seedfile_set.add(new_level_path)
    seedfile_paths.clear()
    seedfile_paths.extend([seedfile for seedfile in seedfile_set])           
```